spoonbill/writers/xlsx.py

```diff
@@ -30,12 +29,22 @@ def __init__(self, workdir, tables, options):
         self.options = options
         self.col_index = collections.defaultdict(dict)
         self.names = {}
+        self.names_counter = defaultdict(lambda: 0)
+
+    def name_check(self, table_name):
```

Comment:
this is completely identical piece of code form csv. probably good idea to combine this somehow